features/form_input.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Form input
+
+@high
+Scenario: Malicious actor inputs executable code to carry out XSS attack
+  Given I am a malicious actor
+  When I visit "/foreign-travel-advice/turkey"
+  And I click on the link "Get email alerts"
+  Then I should see "Create subscription"
+  When I click on the button "Create subscription"
```

Comment:
Quite a few steps here, probably best to move some of the irrelevant details into a couple of simpler steps e.g. "When I add a malicious script to the subscription signup form"